USAGE.md

```diff
@@ -0,0 +1,246 @@
+# Usage
+
+This guide is intended to help you make full use of the DMS.
+
+All the examples here will use Python 3 with the `requests` module, as it
+produces very redable and understandable code. Adaptation into other languages
+should be fairly simple.
+
+## Basic principles
+
+### General operation
+
+DMS mainly has two distinct operations: Uploading a template, and merging it
+with some data to generate a document.
+
+#### Supported engines
+
+To be able to render (generate) a document, you first need to upload a
+template.  DMS currently supports two formats: Mail Merge, and DocxTpl,
+which in turn uses Jinja2 syntax. You can read more about them here:
+
+* [docx-template](https://github.com/elapouya/python-docx-template)
+* [docx-mailmerge](https://github.com/Bouke/docx-mailmerge)
+
+Mail Merge uses placeholders in the office format. This means that the
+placeholders are defined using the features of your office suite, using
```

Comment:
I believe you probably could drop the last part of the sentence (or rewrite it so `using` isn't used twice :smile: ).